BootloaderCorePkg/Library/SmbiosInitLib/SmbiosInitLib.c

```diff
@@ -64,70 +89,70 @@ FindSmbiosType (
   IN  UINT8   Type
   )
 {
-  SMBIOS_STRUCTURE              *TypeHdr;
-  SMBIOS_TABLE_ENTRY_POINT      *SmbiosEntry;
-  UINT8                         *StringPtr;
+  SMBIOS_STRUCTURE        *TypeHdr;
+  SMBIOS_TABLE_ENTRY_POINT    *SmbiosEntry;
+  UINT8             *StringPtr;
 
   SmbiosEntry = (SMBIOS_TABLE_ENTRY_POINT *)(UINTN)PcdGet32 (PcdSmbiosTablesBase);
   if (SmbiosEntry == NULL) {
-    return NULL;
+  return NULL;

```

Comment:
In your changes, the indents don't follow the coding standard.  Please fix it.
If it is the same as old code, please avoid introducing format change.

Reply:
I had fixed it. its due to my editor settings and uploaded new patches. :(